src/test/ui/process/nofile-limit.rs

```diff
@@ -6,7 +6,7 @@
 // dont-check-compiler-stderr
 // only-linux
 // no-prefer-dynamic
-// compile-flags: -Ctarget-feature=+crt-static -Crpath=no
+// compile-flags: -Ctarget-feature=+crt-static -Crpath=no -Crelocation-model=static
```

Comment:
This doesn't apply to the already compiled standard library.

Reply:
As I understand it should be fine to use those artifacts to produce statically linked non position-independent executable.

Reply:
Oh, this is just because PIE is not supported, not because PIC is not supported, right?

Reply:
Static position independent executable to be specific, which is what we would try to build with +crt-static and default relocation model. It requires a different CRT objects, which happen to be unavailable on Void Linux.

Reply:
And this is not necessary in src/test/ui/sanitize/crt-static.rs, src/test/ui/cfg/crt-static-on-works.rs, src/test/run-make/static-pie/Makefile because those are ...? What is special about this test here that it needs this and those don't?

(Found by `rg '[+]crt-static' src/test`, there may be other similar cases).

Reply:
It was the first and only UI test to use static-pie on Linux. Hence this small patch to avoid unnecessary roadblock for contributors using Void Linux. The first UI test you mentioned looks for failure. The second is MSVC only. 

I didn't bother with run-make tests, so I am not sure about third case - it contains version checks to detect support for static-pie, but maybe those should be changed into a more accurate feature test instead.